controllers/fenceagentsremediation_controller.go

```diff
@@ -246,7 +247,7 @@ func isTimedOutByNHC(far *v1alpha1.FenceAgentsRemediation) bool {
 func (r *FenceAgentsRemediationReconciler) updateStatus(ctx context.Context, far *v1alpha1.FenceAgentsRemediation) error {
 	if err := r.Client.Status().Update(ctx, far); err != nil {
 		if !apiErrors.IsConflict(err) {
```

Comment:
Please also log the conflict, it should happen very seldom only when NHC updated the CR for setting the timeout annotation. We are hiding the issue you are resolving with this PR without logging.